### DISCUSSION TITLE
AdGuard Home LXC does not install

### DISCUSSION THREAD

**Discussion by madalin92** [tteck/Proxmox#3501] [Help Needed] [answered]
Hello,

I try to install AdGuard on my Proxmox VE but simply nothing happens:

![image](https://github.com/user-attachments/assets/4d881d93-cba6-4295-b49d-98c764f8afe5)

Am i doing something wrong?

**Comment by tteck** [comment, marked as answer]
🌐 It seems that you're facing a network problem, _probably_ linked to DNS.

**Reply by madalin92** [reply]
I just didn't think about it.

The DNS had remained set to an LXE Pihole which in the meantime no longer exists.

I changed to 1.1.1.1 and everything is ok.

Thank you very much!
